src/icalendar/tests/test_oss_fuzz_errors.py

```diff
@@ -0,0 +1,18 @@
+"""This file collects errors that the OSS FUZZ build has found."""
+from datetime import time
+from icalendar import Calendar
+from icalendar.prop import vDDDLists
+
+import pytest
+
+
+def test_stack_is_empty():
+    """If we get passed an invalid string, we expect to get a ValueError."""
+    with pytest.raises(ValueError):
+        Calendar.from_ical("END:CALENDAR")
+
+
+def test_vdd_list_type_mismatch():
+    """If we pass in a string type, we expect it to be converted to bytes"""
+    vddd_list = vDDDLists([time(hour=6, minute=6, second=6)])
```

Comment:
I am wondering. `to_ical()` - should it always return a string or bytes? Since the library might not have had a refactoring work since Python2.7, I can understand if these get muddled up. I think, we read calendars from bytes... Let's check. I think, this is a quick fix but in the end, we might want to create type annotations and have a common interface. This might be worth creating an issue. I wonder what the other maintainers think.

Reply:
Thanks for pointing that in consistency out!